charts/gateway/production-values.yaml

```diff
@@ -1219,7 +1308,7 @@ redis:
     autoGenerated: true
   image:
     registry: docker.io
-    repository: bitnamilegacy/redis
+    repository: bitnami/redis
```

Comment:
@Gazza7205 This should still point to legacy as per https://github.com/CAAPIM/apim-charts/pull/401 right?

Reply:
I see other parts of #401 already in develop/gateway. Will change to bitnamilegacy